### PR TITLE
Move function to check youtube domain to common class

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch means moving the function to check the youtube domain to the common
utils class. Cause, this function is useful to use in other code.

Refs: #776

### CODE REVIEW

**Review by romandev — CHANGES_REQUESTED** [review]
The method doesn't have a dependency with upstream. Therefore, there is no reason to move it to Banana layer.

**Comment by jdragonbae** [comment]
> The method doesn't have a dependency with upstream. Therefore, there is no reason to move it to Banana layer.

Ack. Thanks.

File: src/org/triple/banana/util/UrlUtil.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import android.text.TextUtils;
+import android.util.Log;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class UrlUtil {
+    private static final String TAG = "UrlUtil";
+
+    public static boolean isYoutubeDomainUrl(String urlString) {
```
Comment (romandev) [inline src/org/triple/banana/util/UrlUtil.java:17]: Make the string non-null

File: src/org/triple/banana/util/UrlUtil.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+package org.triple.banana.util;
+
+import android.text.TextUtils;
+import android.util.Log;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class UrlUtil {
```
Comment (romandev) [inline src/org/triple/banana/util/UrlUtil.java:14]: Make this class's name `YouTubeUtil` or `YouTubeHelper` or other name?

Comment (jdragonbae) [reply src/org/triple/banana/util/UrlUtil.java:14]: `YouTubeUtil` looks good to me!

File: src/org/triple/banana/util/UrlUtil.java (hunk 1 above)
Comment (jdragonbae) [reply src/org/triple/banana/util/UrlUtil.java]: Done.

**Review by romandev — APPROVED** [review]
LGTM

**Pull request merged**